pkg/monitor/dynkeepalived.go

```diff
@@ -306,23 +315,25 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 		case <-done:
 			return nil
 
-		case <-bootstrapStopKeepalived:
+		case APIStateChanged := <-bootstrapStopKeepalived:
 			//Verify that stop message sent successfully
 			for {
-				_, err := conn.Write([]byte("stop\n"))
+				var cmdMsg []byte
+				if APIStateChanged == stopped {
+					cmdMsg = []byte("stop\n")
+				} else {
+					cmdMsg = []byte("reload\n")
+				}
+				_, err := conn.Write(cmdMsg)
 				if err == nil {
-					log.Info("Stop message successfully sent to Keepalived container control socket")
+					log.Info("Command message successfully sent to Keepalived container control socket: %s", string(cmdMsg[:]))
```

Comment:
This isn't formatting correctly. I think you need log.Infof to use the format string.